arbitrum-docs/welcome/get-started.md

```diff
@@ -0,0 +1,80 @@
+---
+title: 'Get started with Arbitrum'
+description: An abbreviated introduction to the Arbitrum suite
+author: symbolpunk
+user_story: As a curious reader, I'd like to be quickly guided towards first steps based on my particular needs.
+content_type: quickstart
+---
+
+<a data-quicklook-from='arbitrum'>Arbitrum</a> is a suite of Ethereum scaling solutions that make it easy to build and use decentralized applications. This document provides a high-level overview of the Arbitrum suite along with onboarding guidance tailored to specific audiences.
+
+
+## The Arbitrum suite
+
+The Arbitrum suite includes the protocols, chains, services, and SDKs that power the Arbitrum ecosystem:
+
+| Component                                                         | Description                                                                                         |
+| ----------------------------------------------------------------- | --------------------------------------------------------------------------------------------------- |
+| [Arbitrum Rollup](/inside-arbitrum-nitro)                         | A **protocol** for scaling Ethereum smart contracts.                                                |
+| [Arbitrum AnyTrust](/inside-anytrust)                             | A **protocol** for scaling Ethereum smart contracts even further, with a mild trust assumption.     |
+| [Arbitrum Nitro](/inside-arbitrum-nitro)                          | The node **software** that codifies the Rollup and AnyTrust protocols.                              |
+| [Arbitrum nodes](/node-running/how-tos/running-a-full-node)       | **Machines** that run Nitro in order to service and/or interact with an Arbitrum chain.             |
+| [Arbitrum One](https://portal.arbitrum.io/?chains=arbitrum-one)   | A public Rollup **chain**.                                                                          |
+| [Arbitrum Nova](https://portal.arbitrum.io/?chains=arbitrum-nova) | A public AnyTrust **chain**.                                                                        |
+| [Arbitrum bridge](https://bridge.arbitrum.io/)                    | Lets you move ERC-20 tokens between Ethereum and Arbitrum.                                          |
+| [Arbitrum Orbit](https://orbit.arbitrum.io/)                      | Lets you run your own Rollup and AnyTrust chains.                                                   |
+| [Stylus](/stylus/stylus-gentle-introduction)                      | Lets you write EVM-compatible smart contracts in Rust and any other language that compiles to Wasm. |
```

Comment:
```suggestion
| [Arbitrum Stylus](/stylus/stylus-gentle-introduction)                      | Lets you write EVM-compatible smart contracts in Rust and any other language that compiles to Wasm. |
```
Not sure, but to follow the same structure as all other concepts 😅

Reply:
Good catch, ty!